Alloy/commands/compile/sourceMapper.js

```diff
@@ -208,8 +208,7 @@ exports.generateSourceMap = function(generator, compileConfig) {
 			sourceType: 'module'
 		});
 	} catch (e) {
-		logger.trace(genMap.code);
-		throw e;
+		U.dieWithCodeFrame('Error parsing code in "' + origFile.filename + '". Unexpected token at line ' + e.loc.line + ' column ' + e.loc.column, e.loc, genMap.code);
```

Comment:
I think the Unexpected token at line was a bad move when I added this to the controller parsing code initially. Maybe we could make use of the `message` property on the error thrown from babel here? I think it should give better context to a user as to what the issue is. So something like

```js
U.dieWithCodeFrame(`Error parsing code in "${origFile.filename}". ${e.message}`, e.loc, genMap.code);
```